modules/jupyter-widget/src/create-deck.js

```diff
@@ -83,10 +83,18 @@ function createDeck({
   customLibraries
 }) {
   let deckgl;
+
+  // First add custom libraries (ensure we have any new class defs before converting props!)
+  try {
+    addCustomLibraries(customLibraries);
+  } catch (error) {
+    // eslint-disable-next-line
+    console.error('pydeck: load of custom library failed', error, customLibraries);
```

Comment:
What exactly would fail here? `loadExternalLibrary` is async.

Reply:
Ah good point. I just wanted to make sure that a failed library load did not bring down the initialization. Will update.

Reply:
@ajduberstein This fix is starting to be a lot trickier than I expected. I just realized that for this to work, I would need to change createDeck to be async, since we can't start feeding props to deck until we have loaded and registered all libraries. Not sure if the ipywidget architecture can handle that.